src/ocm/components/clusterConfiguration/review/ReviewStep.tsx

```diff
@@ -12,15 +19,30 @@ import { useStateSafely } from '../../../../common/hooks';
 import { selectCurrentClusterPermissionsState } from '../../../selectors';
 import ReviewPreflightChecks from './ReviewPreflightChecks';
 import ReviewSummary from './ReviewSummary';
+import { getLimitedFeatureSupportLevels } from '../../../../common/components/featureSupportLevels/utils';
+import { useTranslation } from '../../../../common/hooks/use-translation-wrapper';
+
 import './ReviewCluster.css';
 
 const ReviewStep = ({ cluster }: { cluster: Cluster }) => {
   const { addAlert } = useAlerts();
   const clusterWizardContext = useClusterWizardContext();
   const { isViewerMode } = useSelector(selectCurrentClusterPermissionsState);
   const [isStartingInstallation, setIsStartingInstallation] = useStateSafely(false);
+  const featureSupportLevelData = useFeatureSupportLevel();
+  const { t } = useTranslation();
   const dispatch = useDispatch();
 
+  const hasSupportLevel = React.useMemo<boolean>(() => {
+    let hasSupportLevels = false;
+    try {
+      hasSupportLevels = !!getLimitedFeatureSupportLevels(cluster, featureSupportLevelData, t);
```

Comment:
Do you still need the try-catch after adding the !!
?

Reply:
Yes, the current implementation of `getLimitedFeatureSupportLevels` is that it can throw errors in a few cases and in fact, it will throw one in the case we're handling here (misconfiguration of the dev environment and feature-support-levels missing).

The casting etc doesn't prevent the error from being thrown.